code_of_conduct.md

```diff
@@ -0,0 +1,50 @@
+# Contributor Code of Conduct
```

Comment:
Where did you found this ?
I think it's a good idea but maybe more in wiki, what do you think @paulandrieux
I know Symfony have a [CONTRIBUTING.md](https://github.com/symfony/symfony/blob/master/CONTRIBUTING.md) file where they write how to make PR but it need to be think together before